src/theme/safeTheme.tsx

```diff
@@ -288,7 +288,6 @@ const createSafeTheme = (mode: PaletteMode): Theme => {
             '&. MuiPaper-root': {
               backgroundColor: theme.palette.error.background,
             },
-            border: `1px solid ${theme.palette.error.main}`,
```

Comment:
I suggest we also remove it from the other styles like `standardInfo` and `standardSuccess`

Reply:
Wow I'm blind :D 
Done!